lxd/storage/backend_lxd.go

```diff
@@ -3723,25 +3726,118 @@ func (b *lxdBackend) ImportBucket(projectName string, poolVol *backupConfig.Conf
 	}
 
 	// Validate config and create database entry for restored bucket.
-	bucketID, err := BucketDBCreate(context.TODO(), b, projectName, true, bucket)
+	bucketID, err := BucketDBCreate(b.state.ShutdownCtx, b, projectName, true, bucket)
```

Comment:
These changes to passing in context are unrelated to the commit message and should be in the earlier commit that added this function.